ext/http/01_http.js

```diff
@@ -123,7 +132,7 @@ class HttpConn {
     }
 
     const { 0: streamRid, 1: method, 2: url } = nextRequest;
-    SetPrototypeAdd(this.managedResources, streamRid);
+    SetPrototypeAdd(this.#managedResources, streamRid);
```

Comment:
is the use of `SetPrototype*` functions necessary in this class if `managedResources` is a `SafeSet`? 🤔